linearReg.py

```diff
@@ -0,0 +1,12 @@
+import pandas as pd
+import matplotlib.pyplot as plt
```

Comment:
you do not use this library, don't you?

Reply:
Not anymore, I was using it when I plotted it, but now i don't need it anymore.